core/src/test/scala/com/salesforce/op/stages/impl/feature/TextTokenizerTest.scala

```diff
@@ -220,6 +220,17 @@ class TextTokenizerTest extends OpTransformerSpec[TextList, TextTokenizer[Text]]
         .setDefaultLanguage(Language.English)
     )
   }
+
+  it should "strip html tags and tokenize text correctly " +
+    "inside SmartTextVectorizer via stripHTML flag" in new English {
+    val exampleHTML = "<body>Big ones, small <h1>ones</h1>, some as big as your head</body>".toText
+    val tokenizer = new TextTokenizer[Text](analyzer = TextTokenizer.AnalyzerHtmlStrip)
+    val smartText = new SmartTextVectorizer().setStripHtml(true).setInput(english)
+    val smartTextTokens = smartText.tokenize(exampleHTML).tokens.value
```

Comment:
@leahmcguire @gerashegalov @Jauntbox Could you take a look at this test ? The flag is set correctly (`smartText.getStripHtml` is `true`), but when I call `smartText.tokenize` it was still using the wrong TextAnalyzer. I override the analyzer inside `tokenize` by doing `analyzer: TextAnalyzer = if (getStripHtml) TextTokenizer.AnalyzerHtmlStrip else TextTokenizer.Analyzer`

Reply:
Which TextAnalyzer does it end up using?

Reply:
The normal one, TextTokenizer.Analyzer. If you run the test locally you'll see 

List("big", "ones", "small", "ones", "big", "head") did not contain the same elements in the same (iterated) order as List("body", "big", "ones", "small", "h1", "ones", "h1", "big", "head", "body")
org.scalatest.exceptions.TestFailedException: List("big", "ones", "small", "ones", "big", "head") did not contain the same elements in the same (iterated) order as List("body", "big", "ones", "small", "h1", "ones", "h1", "big", "head", "body")

Reply:
Nvm, turned out it was using the right `TextAnalyzer`, there was just some discrepancy between tokenizing with `SmartTextVectorizer` and tokenizing with a plain `TextTokenizer`

Reply:
What was the difference? And why?